companies/ludwigvonkapff.json

```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "ludwigvonkapff",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Ludwig von Kapff GmbH",
+    "address": "Speicher I\nKonsul-Smidt-Str. 8 J\n28217 Bremen\nDeutschland",
+    "phone": "+49 421 3994317",
+    "fax": "+49 421 3994301",
+    "email": "service@ludwig-von-kapff.de",
```

Comment:
The privacy policy lists a better mail (datenschutz@egfra.de). We should use that instead. Then, we can also set the `suggested-transport-medium` to `email`.